apps/studio/components/layouts/LogsLayout/LogsSidebarMenuV2.tsx

```diff
@@ -366,9 +367,27 @@ export function LogsSidebarMenuV2() {
           />
         )}
         {savedQueries.map((query) => (
-          <SavedQueriesItem item={query} key={query.id} />
+          <SavedQueriesItem item={query as any} key={query.id} /> // kemal: i know, i know, temp any fix.
         ))}
       </SidebarCollapsible>
+
+      <FeaturePreviewSidebarPanel
+        className="mx-4 mt-4"
+        title="Capture your logs"
+        description="Send logs to your preferred observability or storage platform."
+        illustration={
+          <div className="flex items-center gap-4">
+            {LOG_DRAIN_TYPES.map((type) =>
+              React.cloneElement(type.icon, { height: 20, width: 20 })
+            )}
+          </div>
+        }
+        actions={
+          <Button asChild type="default">
+            <Link href={`/project/${ref}/settings/log-drains`}>Go to Log Drains</Link>
+          </Button>
+        }
+      />
```

Comment:
I believe this component is for the old logs, right? If so, this feedback can likely be ignored since I understand we’re moving away from it:

- This panel looks a bit funny next to the centered text in the aside above it
- Maybe a `Separator` above this would help make it not look like a child of _Queries_

Reply:
P.S. is this `React.cloneElement` the reason for the peculiar `import React, { useState } from 'react'` import?

Seems like a funny bit of syntax to map through icons, but I’ll leave that to frontender(s) to assess. It’s more likely that I need to brush up.